src/components/Header/Header.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Consider removing the eslint-disable comment and fixing the accessibility issue by adding a label to the button. ♿

Reply:
done